### PR TITLE
Add computer science to protected terms

### PR DESCRIPTION
**Pull request by Luggas4you**
This fixes https://github.com/JabRef/jabref-issue-melting-pot/issues/236.

grafik

### Mandatory checks
- [x] Change in `CHANGELOG.md` described in a way that is understandable for the average user (if applicable)
- [ ] Tests created for changes (if applicable)
- [x] Manually tested changed features in running JabRef (always required)
- [x] Screenshots added in PR description (for UI changes)
- [ ] [Checked developer's documentation](https://devdocs.jabref.org/): Is the information available and up to date? If not, I outlined it in this pull request.
- [ ] [Checked documentation](https://docs.jabref.org/): Is the information available and up to date? If not, I created an issue at <https://github.com/JabRef/user-documentation/issues> or, even better, I submitted a pull request to the documentation repository.

### CODE REVIEW

**Comment by Luggas4you** [comment]
Should I write the addition in the CHANGELOG.md?

**Comment by Siedlerchr** [comment]
Yep, changelog and have a look at the failing tests

**Comment by Luggas4you** [comment]
> Yep, changelog and have a look at the failing tests -> LocalizationConsistencyTest

I fixed the problems of the test [Add Computer science to en.properties](https://github.com/JabRef/jabref/pull/10222/commits/23e86593052f8de8441acfd98515c59bacb4b260) and added a description to the CHANGELOG.md [Add change in CHANGELOG.md](https://github.com/JabRef/jabref/pull/10222/commits/7602a37a36aa416eb38d461517cb20db2076aead)

**Comment by Siedlerchr** [comment]
Ah the ProtectedTermsFormatter test is now failing because it recognizes the new terms

File: src/main/resources/protectedterms/computer_science.terms [hunk 1]
```diff
@@ -0,0 +1,5 @@
+JabRef
```
Comment (koppor) [inline src/main/resources/protectedterms/computer_science.terms:1]: This list should be sorted alphabetically. `BibTeX` should be added, too.

Comment (Luggas4you) [reply src/main/resources/protectedterms/computer_science.terms:4]: I fixed the sortation and added `BibTeX` to the list.

**Review by Siedlerchr — APPROVED** [review]

**Pull request merged**